samples/saga/migration/Core_6/Server/TestSaga.cs

```diff
@@ -0,0 +1,92 @@
+﻿#region Toggle
+
+//#define MIGRATION
+
+#endregion
+
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+#region Header
+
+#if NEW
+[NServiceBus.Persistence.Sql.SqlSaga(correlationProperty: "SomeId")]
+#endif
+
+public class TestSaga :
+        Saga<TestSaga.TestSagaData>,
+        IHandleMessages<ReplyFollowUpMessage>,
+        IHandleMessages<CorrelatedMessage>,
+        IHandleTimeouts<TestTimeout>,
+#if MIGRATION && !NEW
+    IHandleMessages<StartingMessage>,
+    IAmStartedByMessages<DummyMessage>
+#else
+        IAmStartedByMessages<StartingMessage>
+#endif
+
+    #endregion
+
+{
+#if MIGRATION && !NEW
+    public Task Handle(DummyMessage message, IMessageHandlerContext context)
```

Comment:
let's add the comment about this being required for validation reasons comment from the doco page here as well